week-1/exercises/initials.js

```diff
@@ -4,3 +4,5 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string in upper case to form the user's initials
 // Log the variable in each case
+var initials=  `${firstName.charAt(0).toUpperCase()} ${middleName.charAt(0).toUpperCase()}${lastName.charAt(0).toUpperCase()}`;
```

Comment:
You have used the correct methods. Just double-check the output. Can we make it look better?